tt_metal/include/compute_kernel_api/eltwise_unary/remainder.h

```diff
@@ -0,0 +1,46 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+
+#include "compute_kernel_api/common_globals.h"
+#ifdef TRISC_MATH
+#include "llk_math_eltwise_unary_sfpu_remainder.h"
+#define MAIN math_main()
+#define MATH(x) x
+#else
+#define MATH(x)
+#endif
+
+
+
+namespace ckernel {
+
+/**
+ * Performs element-wise modulus computation on input x by y , where x is each element of a tile
+ * in DST register at index tile_index. The input can be of float data type. The value is provided as const param0 The DST register buffer must be in
+ * acquired state via *acquire_dst* call. This call is blocking and is only
+ * available on the compute engine.
+ *
+ * Return value: None
+ *
+ * | Argument        | Description                                                                | Type     | Valid
+ * Range                                           | Required |
+ * |-----------------|----------------------------------------------------------------------------|----------|-------------------------------------------------------|----------|
+ * | idst            | The index of the tile in DST register buffer to perform the computation on | uint32_t | Must be
+ * less than the size of the DST register buffer | True     | | param0          | The value the output is if the input
+ * is greater than 0                     | uint32_t |                                                       | True     |
```

Comment:
Add descriptions of the params to the table. The table is also formatted weirdly (in code, not sure about how it actually shows in docs). Is it possible to fix it so it's easier to read

Reply:
Sure we will see how it reflect in docs and modify it

Reply:
We have checked and it seems to be aligned properly,
<img width="1429" alt="Screenshot 2024-06-14 at 9 10 04 AM" src="https://github.com/tenstorrent/tt-metal/assets/156493059/3d74b199-0e13-4a51-82ed-3861b7b05c51">

Reply:
Can you add descriptions for param0 and param1 to the table?

Reply:
<img width="1302" alt="Screenshot 2024-06-17 at 4 36 34 PM" src="https://github.com/tenstorrent/tt-metal/assets/138196495/27de1172-6ad7-4318-b0b9-2900d9b3b872">
We've updated the code @tt-aho